crates/api/src/main.rs

```diff
@@ -94,6 +108,18 @@ async fn main() {
     let s3_client = s3::Client::new(s3_endpoint, Some(Box::new(creds)), None, None)
         .expect("Failed to connect to S3");
 
+    let email_creds = Credentials::new(
+        std::env::var("SMTP_USERNAME").unwrap(),
+        std::env::var("SMTP_PASSWORD").unwrap()
+    );
+
+    let mailer = SmtpTransport::starttls_relay(
```

Comment:
Could you make this like the previous Python codebase where if SMTP_HOST is not present mailer would be None and codes would be debugging in console?